typescript/src/languages/en-US/commands/fun.json

```
@@ -193,6 +193,18 @@
 			"Skyra"
 		]
 	},
+	"thinkDescription": "Think, Mark!",
+	"thinkExtended": {
+		"usages": [
+			"User"
+		],
+		"extendedHelp": "\"Think, Mark!\" is a meme from the TV show Invincible, the rest... no, that would be spoilers.",
+		"reminder": "This command contains animated blood, which may not be suitable for all audiences.",
```

```suggestion
		"reminder": "This command contains cartoon blood, which may not be suitable for all audiences.",
```

animated in this context implies movement, but it's just a png.